puppet/fabfile.py

```diff
@@ -128,6 +170,11 @@ def puppet_apply(dry_run='no'):
             " "+manifest_to_run+" "
             )
 
```

Comment:
We did talk about replacing our current puppet setup with... Ansible, was it? That may not be high-enough priority to do instead of this (in fact it probably isn't) but it may mean that we can just use `restart_uber_service()` for now.

Separately, does putting the upgrade here mean DB migrations get auto-run? Is that a good idea?